packages/legend-studio-app/src/index.tsx

```diff
@@ -16,12 +16,13 @@
 
 import { Studio } from '@finos/legend-studio';
 import { DSLText_Preset } from '@finos/legend-studio-preset-dsl-text';
+import { EFJSONSchema_Preset } from '@finos/legend-studio-preset-external-format-json-schema';
```

Comment:
need to add json-schema package to the `pacakge.json` and `tsconfig.json` of this package.